project/Dependencies.scala

```diff
@@ -57,7 +57,10 @@ object Dependencies {
     "org.flywaydb" % "flyway-core" % flyway
   )
 
-
+  lazy val zookeeperDeps = Seq(
+    "org.apache.curator" % "curator-framework" % curator excludeAll excludeZookeeper,
```

Comment:
The build seem to fail due to Guava dependency issue.
Curator is bringing guava-20.0, which seem to break cassandraunit at least (currently cassandraunit uses guava-18.0).
As one of the fixes, there should be new exclusion rule for curator:
```
val excludeGuava = ExclusionRule(organization = "com.google.guava")
```
For both test and framework:
```
"org.apache.curator" % "curator-framework" % curator excludeAll(excludeZookeeper, excludeGuava),
..
"org.apache.curator" % "curator-test" % curatorTest % Test excludeAll(excludeGuava)
```

Reply:
Thank you for the feedback, I added a commit that addresses this dependency issues.
Tests still fail with some sbt dependency resolving issue however, will look into this.